wicket/src/screens/component.rs

```diff
@@ -240,6 +235,11 @@ impl ComponentScreen {
         vec![Action::Redraw]
     }
 
+    fn resize(&mut self, width: u16, _height: u16) {
+        self.rack_screen_button_state.rect.x =
+            width - ScreenButtonState::width();
```

Comment:
Pathological case, but this panics if the screen is thinner than `ScreenButtonState::width()`. Do we care, or is that already prevented by some other means?

Reply:
It basically doesn't matter IMO. I could use a saturating add, but the width of that button is around 7 characters. We really need to set a minimum width of the terminal to at least 40 characters, but more like 72. In short, it just shouldn't happen and I don't care if it does right now. I'm happy to provide some preliminary checks on startup so we fail with a good error message in a future PR, but this probably doesn't warrant any changes.